src/main/java/Duke.java

```diff
@@ -6,5 +9,78 @@ public static void main(String[] args) {
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        String intro = "Hello! I'm Duke \n" +
+                    "What can I do for you? \n";
+        System.out.println(intro);
+        List<Task> ls = new ArrayList<>();
+        while(true) {
+            try { 
+                Scanner sc = new Scanner(System.in);
+                Task n = new Task(sc.nextLine());
```

Comment:
```suggestion
                Task task = new Task(sc.nextLine());
```
For coding standard, to avoid single letter variables unless as scratch variables